src/coreclr/jit/lsra.cpp

```diff
@@ -2837,8 +2844,17 @@ regNumber LinearScan::allocateReg(Interval*    currentInterval,
             }
         }
     }
+
     assignPhysReg(availablePhysRegRecord, currentInterval);
     refPosition->registerAssignment = foundRegBit;
+
+#ifdef TARGET_ARM64
```

Comment:
Meta question on the `#ifdef`s: do we anticipate using this mechanism on other targets? Usually such things that are really target-independent but are not used on all targets are wrapped in another `#define`, e. g. `FEATURE_RARE_CONSECUTIVE_REGISTERS` (to give a hint it doesn't work that way for `TARGET_ARM/TYP_DOUBLE`...).

Reply:
That's a fair point. I will add something.